index.js

```diff
@@ -83,16 +83,22 @@ const queryDns = (version, options) => {
 
 	const socket = dns({
 		retries: 0,
+		maxQueries: 1,
```

Comment:
> Another problem is the default maxQueries (10000) of dns-socket which is overkill because we just do one query at the same time

If we only do one query at the time, how does setting this change anything?

Reply:
`dns-socket` internally creates an empty array with 10000 empty values, later it may or not loop each entry searching for just one query and anyway is a waste of memory.
Check this line: https://github.com/mafintosh/dns-socket/blob/master/index.js#L31